packages/babel-core/test/config-ts.js

```diff
@@ -0,0 +1,82 @@
+import { loadPartialConfigSync } from "../lib/index.js";
+import path from "path";
+import { fileURLToPath } from "url";
+import { createRequire } from "module";
+import semver from "semver";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const require = createRequire(import.meta.url);
+
+// We skip older versions of node testing for two reasons.
+// 1. ts-node does not support the old version of node.
+// 2. In the old version of node, jest has been registered in `require.extensions`, which will cause babel to disable the transforming as expected.
+
+describe("@babel/core config with ts [dummy]", () => {
+  it("dummy", () => {
+    expect(1).toBe(1);
+  });
+});
+
+semver.gte(process.version, "12.0.0")
+  ? describe
+  : describe.skip("@babel/core config with ts", () => {
+      it("should work with simple .cts", () => {
+        const config = loadPartialConfigSync({
+          configFile: path.join(
+            __dirname,
+            "fixtures/config-ts/simple-cts/babel.config.cts",
```

Comment:
Just curious, does `new URL("./fixtures/config-ts/simple-cts/babel.config.cts", import.meta.url)` work?

Reply:
I'm not sure, I've encountered a lot of `URL` with windows causing special exceptions, so I always use `path`. And they look just as readable.